arch/ecmwf/aurora/nec/4.0.0/toolchain.cmake

```diff
@@ -0,0 +1,89 @@
+
+####################################################################
+# COMPILER
+####################################################################
+
+include( /opt/nec/ve/share/cmake/toolchainVE.cmake )
+
+
+set( ECBUILD_FIND_MPI ON )
+
+####################################################################
+# Enviroment Variables
+####################################################################
+set(NMPI_ROOT /opt/nec/ve/mpi/2.23.0)
```

Comment:
Is this the same on all NEC VE machines, or should this maybe be exported as an environment variable in env.sh and then picked up in the toolchain file?

Reply:
No, it's not. I think environment variable is the better idea for it.